src/StickyMenu.php

```diff
@@ -243,6 +244,10 @@ public function body_class( $classes ) {
 			$classes[] = 'js-sticky-navigation';
 		}
 
+		if ( get_theme_mod( 'sticky_menu_layout', $this->default_settings['sticky_visibility'] ) == 'all' ) {
```

Comment:
@krumpak the setting name is still "layout", this should be changed to "visibility"